packages/terra-functional-testing/src/terra-cli/wdio/test-runner.js

```diff
@@ -102,6 +114,7 @@ class TestRunner {
             formFactor,
```

Comment:
late to the party, but dang, i like the test runner architecture here. i'll rearrange the full stack test runner to mirror this format. it's much cleaner